### ISSUE TITLE
AttributeError: PDFDictionary instance has no attribute 'decodedStream'                                      

### ISSUE THREAD

**Issue by jcppkkk** [cuckoosandbox/cuckoo#1994]
hash: 36083e4094f69fe0366aedf89171d3432c80edbb5e38c2b6d68ab0014f1d112b9440fd3e1cda644432495c472ee2b0c1

```
Traceback (most recent call last):                                                                           
  File "/usr/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap                              
    self.run()                                                                                               
  File "/usr/lib/python2.7/multiprocessing/process.py", line 114, in run                                     
    self._target(*self._args, **self._kwargs)                                                                
  File "cuckoo/cuckoo/misc.py", line 161, in _worker                                   
    conn.send(func(*args, **kwargs))                                                                         
  File "cuckoo/cuckoo/processing/static.py", line 1034, in _pdf_worker                 
    return PdfDocument(filepath).run()                                                                       
  File "cuckoo/cuckoo/processing/static.py", line 721, in run                          
    js = self.get_javascript(obj, f, version)                                                                
  File "cuckoo/cuckoo/processing/static.py", line 624, in get_javascript               
    "orig_code": obj.object.decodedStream,                                                                   
AttributeError: PDFDictionary instance has no attribute 'decodedStream'                                      
```

**Comment by RicoVZ** [comment]
Hi jcppkkk,

I am currently working on some of these static processing issue. I however have not been able to find a sample triggering this error, and I cannot find the sample that matches the hash you have posted.

Would it be possible for you to post the actual file?

**Comment by jcppkkk** [comment]
Hi @RicoVZ , sorry i also cannot find the sample now. let run cuckoo with un-patched code for few days, maybe i can find other samples.

**Comment by RicoVZ** [comment]
Hi jcppkkk,

I have changed/fixed the way the Javascript is retrieved, so this error cannot occur anymore.

Thanks a lot for reporting this issue and putting time into retrieving samples, it is much appreciated.

**Comment by jcppkkk** [comment]
@RicoVZ That's great news, thank you :smiley:

**Comment by jcppkkk** [comment]
It should be fixed by 434d8c9
